moment_kinetics/test/fokker_planck_tests.jl

```diff
@@ -108,28 +107,19 @@ function runtests()
                 end
             end
             
-            # fill fc with fvpavperp
-            ravel_vpavperp_to_c!(fc,fvpavperp,vpa.n,vperp.n)
-            ravel_c_to_vpavperp!(fvpavperp_test,fc,nc_global,vpa.n)
-            @. fvpavperp_err = abs(fvpavperp - fvpavperp_test)
-            max_ravel_err = maximum(fvpavperp_err)
-            @serial_region begin
-                if print_to_screen 
-                    println("max(ravel_err)",max_ravel_err)
-                end
-                @test max_ravel_err < 1.0e-15
-            end
+            # Make 1d views
+            fc = vec(fvpavperp)
+            d2fc_dvpa2 = vec(d2fvpavperp_dvpa2_num)
+            d2fc_dvperp2 = vec(d2fvpavperp_dvperp2_num)
+
             #print_vector(fc,"fc",nc_global)
             # multiply by KKpar2D and fill dfc
             mul!(dfc,KKpar2D_with_BC_terms_sparse,fc)
             mul!(dgc,KKperp2D_with_BC_terms_sparse,fc)
-            # invert mass matrix and fill fc
-            fc = lu_obj_MM \ dfc
-            gc = lu_obj_MM \ dgc
+            # invert mass matrix
+            d2fc_dvpa2 .= lu_obj_MM \ dfc

```

Comment:
In other places \ has been replaced by ldiv!, would be good to be consistent here (or explain why it isn't!)